src/app/project/comments/comments.component.html

```diff
@@ -44,13 +44,10 @@ <h3>
                 <label>Partner First Nation(s)</label>
                 <p class="value">{{project?.partner || "-" }}</p>
               </div>
-              <div *ngIf="projectFieldType(project?.agreements) ==='string'" class="agreements">
+              <div *ngIf="project?.agreements?.length > 0" class="agreements">
```

Comment:
Back in the day, projects only had a single agreement which was a string. We kept some logic in place in case that's still the case (but we should have left a comment for it).

We may want to retain that logic, just in case. The other option would be to log in to production LUP and make sure none of the projects are using a string as their agreement. We can chat further about how to do that if you want to go that route